sympy/solvers/ode.py

```diff
@@ -277,10 +277,10 @@
 #: ``best``, and ``all_Integral`` meta-hints should not be included in this
 #: list, but ``_best`` and ``_Integral`` hints should be included.
 allhints = (
-    "separable",
-    "1st_exact",
     "1st_linear",
     "Bernoulli",
+    "separable",
+    "1st_exact",
```

Comment:
I believe those are the same, because `F(x)` is an arbitrary function of `x`, so `exp(F(x))*exp(-2*x))` is also just an arbitrary function of `x` (it's similar to simplifying `exp(2*C1)` to just `C1`). 

By the way, mark your code blocks as code with backticks in GitHub comments (see https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown). Otherwise it renders out the *s and makes it hard to read.